operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -1753,6 +1838,15 @@ def isSSLListenPortEnabled(ssl, domain):
       enabled = True
   return enabled
 
+def getSSLPortIfEnabled(server, domain):
+  ssl = getSSLOrNone(server)
+  ssl_listen_port = None
+  if ssl is not None and ssl.isEnabled():
+    ssl_listen_port = getRealSSLListenPort(server, ssl.getListenPort())
+  elif ssl is None and isSecureModeEnabledForDomain(domain):
```

Comment:
what if ssl is not None but disabled?

Reply:
the it will return None

Reply:
But isSecureModeEnabledForDomain is true. Not sure what the expected behavior for this is going to be.

Reply:
Yes, I didn't change the existing logic.  This PR will not touch secure domain configuration